examples/pytorch/graphsage/train_full.py

```diff
@@ -14,40 +14,7 @@
 import dgl
 from dgl import DGLGraph
 from dgl.data import register_data_args, load_data
```

Comment:
`nx`, `nn`, `DGLGraph`, and `register_data_args` not used

Reply:
Thanks for catching these. In fact, register_data_args is used. I removed nx,nn and DGLGraph

Reply:
Also, I cannot find any documentation for functions like: dgl.data.load_data  and dgl.data.register_data_args. Can you please point where I can find the descriptions for these functions in dgl library?

If they have not been documented, I am more inclined to say avoiding use them as much as possible. I will remove&test within my next update.

Reply:
I don't know doc for them as well. I also think it's a bit strange and inconsistent to have them only for `Cora`, `Citeseer`, and `Pubmed`.

Reply:
Got it. yeah... like you mentioned, using dgl.data instead of load_data can circumvent it. We should remove all load_data/register_data_args in the rest of the examples as there is no supporting doc. Ideally, it would be nice to completely deprecate load_data/register_data_args, since it seems can be fully covered by dgl.data (right?)

Reply:
Yes, but these are long-lasting APIs. So I'll just remove their usage in the examples for now in case deprecating them breaks too many codebases.